### ISSUE TITLE
Values of some parameters 

### ISSUE THREAD

**Issue by fansariadeh** [tkipf/pygcn#59]
In the module named "MODELS" :

class GCN(nn.Module):
    def __init__(self, nfeat, nhid, nclass, dropout):

but the values of these paramters are not specified in the code. I am confused how the code gets these values! nfeat, nhid, nclass

would be great helping me out

**Comment by Jaborie203** [comment]
Please check line 45-48 in "train.py": 
model = GCN(nfeat=features.shape[1],
            nhid=args.hidden,
            nclass=labels.max().item() + 1,
            dropout=args.dropout)

**Comment by fansariadeh** [comment]
> Please check line 45-48 in "train.py":
> model = GCN(nfeat=features.shape[1],
> nhid=args.hidden,
> nclass=labels.max().item() + 1,
> dropout=args.dropout)

Dear Jaborie, I appreciate your response. I was wondering do you happen to know in layers module:
class GraphConvolution(Module):
    def __init__(self, in_features, out_features, bias=True):

Where the values of these parameters are fed into the system? They have not been mentioned anywhere in the code: in_features, out_features

Thank you in advance
